README.md

```diff
@@ -2501,6 +2501,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Vrukshali Torawane](https://github.com/Vrukshali-26)
   - [VulcanWM](https://github.com/VulcanWM)
   - [Vurugonda Kalyan](https://github.com/kalyan-vurugonda)
+  - [Vikrant Kumar](https://github.com/vikrant-sudo)
```

Comment:
Please add your name in line 2471, after Vikhyath HRS.
Let us know if you need any help doing that. 🙂

Reply:
Pardon the question, are the names supposed to appear in alphabetical order? 

Genuinely curious because I noticed the section above appears to have a repetitive V pattern.

Reply:
In order to maintain consistency and ease of navigation, please arrange your name alphabetically. This includes meticulously considering each letter's position, starting from the first and extending to the second, third, and so on.
Thanks!

---
**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Yes I did. Thanks for the further clarification.